src/openms/source/APPLICATIONS/TOPPBase.cpp

```diff
@@ -2586,6 +2587,7 @@ namespace OpenMS
     for (int i = argc - 1; i > 0; --i)
     {
       String arg = argv[i];
+      if (arg.hasPrefix("--")) arg = arg.substr(1); // replace '--' by '-' such that it's found in param_map
```

Comment:
yep. I was afraid to break backwards compatibility, since it could be quite annoying for people to change all their scripts etc...